Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,27 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    def installation_file? f
+      if File.directory?(f)
+        false
+      else
+        f = File.basename(f)
+        !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
+      end
+    end
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? { |f| installation_file? f }
+      problem <<-EOS.undent
+        The installation seems to be empty. Please ensure the prefix
+        is set correctly and expected files are installed.
+        The prefix configure/make argument may be case-sensitive.
+      EOS
+    end
```

Comment:
With no other modifications to the ` audit ` file I'm getting the strangest unrelated error on that chunk of code:

```ruby
  def audit_prefix_has_contents
    return unless formula.prefix.directory?

    Dir.glob("#{formula.prefix}/**/*").each do |file|
      path = Pathname.new file
      next if path.directory?
      basename = path.basename
      return if !Metafiles.copy? basename
      return if !%w[.DS_Store INSTALL_RECEIPT.json].include? basename
    end

    problem <<-EOS.undent
      The installation seems to be empty. Please ensure the prefix
      is set correctly and expected files are installed.
      The prefix configure/make argument may be case-sensitive.
    EOS
  end
```

```
~> brew audit osh
Error: undefined method `downcase' for #<Pathname:AUTHORS>
Please report this bug:
    http://git.io/brew-troubleshooting
/usr/local/Library/Homebrew/metafiles.rb:18:in `copy?'
/usr/local/Library/Homebrew/cmd/audit.rb:669:in `block in audit_prefix_has_contents'
/usr/local/Library/Homebrew/cmd/audit.rb:665:in `each'
/usr/local/Library/Homebrew/cmd/audit.rb:665:in `audit_prefix_has_contents'
/usr/local/Library/Homebrew/cmd/audit.rb:712:in `audit'
```